coopiteasy_custom/data/cron.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2017-2018 Coop IT Easy
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <data noupdate="0">  <!-- fixme-->
+        <record id="cie_fix_account_analytic_line_so_line" model="ir.cron">
+            <field name="name">cie fix account_analytic_line so_line</field>
+            <field name="active" eval="False"/>
+<!--            <field name="user_id" ref="base.user_root"/>-->
```

Comment:
if you don't plan to define a value for it. remove this line

Reply:
kay babe